devices/tuya.js

```diff
@@ -3073,4 +3073,21 @@ module.exports = [
             await reporting.bind(endpoint, coordinatorEndpoint, ['genPowerCfg', 'msTemperatureMeasurement', 'msRelativeHumidity']);
         },
     },
+    {
+        fingerprint: [{modelID: 'TS0601', manufacturerName: '_TZE200_0u3bj3rc'}],
+        model: 'Human_presence_sensor',
```

Comment:
Is there any other model number written on the device? If not please use `TS0601_human_presence_sensor`.

Reply:
finished, thanks